django-data/image/submissions/tests/test_view_validation_summary.py

```diff
@@ -0,0 +1,49 @@
+from django.test import Client, TestCase
+from django.urls import resolve, reverse
+
+from common.tests import GeneralMixinTestCase, OwnerMixinTestCase
+
+from ..views import SubmissionValidationSummaryView
+
+
+class SubmissionValidationSummaryViewTest(TestCase):
```

Comment:
This should inherit from `common.tests` `GeneralMixinTestCase` and `OwnerMixinTestCase`. Take a look to a detail view test as in `submissions.test.test_view_submissions`

Reply:
Done